### PR TITLE
Set Up Associations

### PR DESCRIPTION
**Pull request by orozCoding**
In this branch, I created the remaining classes for the school library and created the associations between them.

- Created the Classroom class
- Created the has-many/belongs-to relationship between Classroom and Student
- Created the Book class
- Created the Rental class
- Created the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Book class
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```
Comment (bimbolabuari) [inline book.rb]: Getters and Setters for `rentals` should be present, Kindly add the `rentals` to attr_accessor. 👍

File: person.rb [hunk 2]
```diff
@@ -8,9 +8,10 @@ def initialize(age, name = 'Unknown', parent_permission = 'true')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (bimbolabuari) [inline person.rb]: Getters and Setters for `rentals` should be present, Kindly add the `rentals` to attr_accessor. 👍

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Rental class
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```
Comment (bimbolabuari) [inline rental.rb]: Getters and Setters for `@book` and `@person` instance variables should be present. Kindly add them to the attr_accessor. 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @orozCoding, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Set up associations between classes and objects.
- Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
Thank you @bimbolabuari

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @orozCoding, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**